src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -123,121 +75,60 @@ Safari and Chrome add a tap highlight color which can be prevented with the
 Internet Explorer on Windows Phone has a similar behavior, but is suppressed
 via a meta tag:
 
-
     <meta name="msapplication-tap-highlight" content="no">
-    
 
-##### Override FirefoxOS Button State Styles
+Firefox has two side effects to handle.
 
-The Firefox `-moz-focus-inner` pseudo class includes an outline on touchable elements.
-You can remove this outline by setting the `border: 0`.
+The `-moz-focus-inner` pseudo class which adds an outline on
+touchable elements, which you can remove by setting `border: 0`.
 
-If you are using a `<button>` element, you get a gradient applied to your
-button which you can remove by setting `background-image: none`.
+If you are using a `<button>` element on Firefox, you get a gradient
+applied, which you can remove by setting `background-image: none`.
 
 <pre class="prettyprint">
 {% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/states-example.html" region_tag="ff-specific" adjust_indentation="auto" %}
 </pre>
 
-##### Override Element Outline in Focus State
-
-Suppress the outline color when an element is focused using `outline: 0`.
-
-
-    .btn:focus {
-      outline: 0;
-    
-      // Add replacement focus styling here (i.e. border)
-    }
-    
-
-#### Disable user-select on UI which Responds to Touch
+Caution: Only suppress the default styles mentioned above if you have pseudo classes for `:hover`, `:active` and `:focus`!
 
-Some mobile browsers will select text if the user long presses on the screen.
-This can result in a bad user experience if the user accidentally presses down
-on a button for too long. You can prevent this from happening using the
-`user-select` CSS property.
+### Disabling user-select
 
+If you are crafting an experience for touch you may want to make parts of
+your UI which can't have text select, elements like buttons or form input.
+You can do this by setting  the `user-select` CSS property, but beware that
+doing this on content users want to select can be *extremely* infuriating
+for users.
 
-    -moz-user-select: none;
-    -webkit-user-select: none;
```

Comment:
FYI, we only just [shipped unprefixed user-select](https://bugs.chromium.org/p/chromium/issues/detail?id=605419) in Chrome 54 - so not quite stable yet.  Fine with me though.